.github/workflows/prerelease_linux.yml

```diff
@@ -105,7 +105,7 @@ jobs:
           password: ${{ env.DOCKER_HUB_PASSWORD }}
 
       - name: Releasing linux packages
```

Comment:
pre-release?

Reply:
good question, is it release to staging or just pre-release 🤔